api/src/Entity/Checklist.php

```diff
@@ -71,7 +71,7 @@ class Checklist extends BaseEntity implements BelongsToCampInterface, CopyFromPr
     #[ApiProperty(example: '/camps/1a2b3c4d')]
     #[Groups(['read', 'create'])]
     #[ORM\ManyToOne(targetEntity: Camp::class, inversedBy: 'checklists')]
-    #[ORM\JoinColumn(nullable: false, onDelete: 'cascade')]
+    #[ORM\JoinColumn(onDelete: 'cascade')]
```

Comment:
Do we want a validation which allow camp to be null only if `isPrototype` is true? And it must be null if `isPrototype` is false?

Otherwise one could create a prototype checklist in a camp, even if this person is not an admin.